quickstarts/consolequickstart-install-cryostat.yaml

```diff
@@ -0,0 +1,144 @@
+          about this topic.
+        instructions: >-
+          #### Verify that **cert-manager Operator for Red Hat OpenShift** is installed:
+
+            - From the **Administrator** perspective, go to **Installed Operators** from the [Operators]{{highlight qs-nav-operators}} section of the navigation. 
+            - The cert-manager Operator for Red Hat OpenShift should be shown in the list of Operators
+            - In the **Status** column, do you see the status of **cert-manager Operator for Red Hat OpenShift** as **Succeeded**?
+
+    - title: Install the Red Hat build of Cryostat Operator
+      description: >
+        To install the Red Hat build of Cryostat operator:
+       
+        1. From the **Administrator** perspective, go to the **OperatorHub** from the [Operators]{{highlight qs-nav-operators}} section of the navigation. 
+        
+        1. In the **Filter by keyword** field, type `Red Hat build of Cryostat`.
+        
+          **Note:** If the tile has an **Installed** label, the Operator is already installed. Proceed to the next task to create an instance of Cryostat.
+        
+        1. Click the **Red Hat build of Cryostat** tile to open the Operator details.
+        
+        1. In the Red Hat build of Cryostat panel, click **Install**. The **Install Operator** page opens with default values.
+        
+        1. Select the default channel `stable`.
+        
+        1. Click **Install** to start the Operator installation. A page opens that shows the various installation steps and displays a message when the Operator is installed.
+        
+        1. Click **View installed Operators in Namespace** to open the **Installed Operators** page.
+
+      review:
+        failedTaskHelp: >-
+          This task isn’t verified yet. Try the task again, or [read
+          more](https://access.redhat.com/documentation/en-us/red_hat_build_of_cryostat/2/html/installing_cryostat/assembly_installing-cryostat_cryostat)
+          about this topic.
+        instructions: >
+          #### Verify that the Red Hat build of Cryostat is installed:
+          
+          - From the **Administrator** perspective, go to **Installed Operators** from the [Operators]{{highlight qs-nav-operators}} section of the navigation. 
+
+          - The Red Hat build of Cryostat should be shown in the list of Operators.
+
+          - In the **Status** column, do you see the status of the Red Hat build of Cryostat as `Succeeded`?
+    - title: Create a Cryostat instance
+      description: >
+        To create a Cryostat instance:
+
+        1. From the **Administrator** perspective, go to **Installed Operators** from the [Operators]{{highlight qs-nav-operators}} section of the navigation. 
+        
+        1. In the **Search by name** field type and search for the Red Hat build of Cryostat. If the operator is not found, ensure that you have selected the project where the Cryostat Operator was installed. 
```

Comment:
Makes sense to me since it's referring to entering text.

```suggestion
        1. In the **Search by name** field type and search for the `Red Hat build of Cryostat`. If the operator is not found, ensure that you have selected the project where the Cryostat Operator was installed. 
```